test/plugins/code/insert.js

```diff
@@ -0,0 +1,16 @@
+/** @jsx h */
+
+export default change => change.insertCode()
+
+export const input = <value>
+    <document>
+        <p><cursor/>Some code</p>
+    </document>
```

Comment:
Move `<cursor/>` inside text to check that it's being split properly. Also add a test in which code is added into an expanded selection (I think that in such case selected text should be moved into new code block).